ROADMAP.md

```diff
@@ -1,30 +1,32 @@
 
-### Current Roadmap
+## Current Roadmap
 The following table includes the current roadmap for Carvel. If you have any questions or would like to contribute to Carvel, please attend a [community meeting](https://carvel.dev/community/) to discuss with our team. If you don't know where to start, we are always looking for contributors that will help us reduce technical, automation, and documentation debt.
 Please take the timelines & dates as proposals and goals. Priorities and requirements change based on community feedback, roadblocks encountered, community contributions, etc. If you depend on a specific item, we encourage you to attend community meetings to get updated status information, or help us deliver that feature by contributing to Carvel.
 
-|Theme|Description|Timeline|
-|---|---|---|
-| Package Author Improvements | **[ytt]** [Schema Validations](https://hackmd.io/pODV3wzbT56MbQTxbQOOKQ#Part-7-Validating-Documents) Experimental: Configuration authors can specify the valid range or format of the data values. | Late July |
-| Provide Stability | **[kapp-controller]** [Surface namespaces for deployed resources](https://github.com/vmware-tanzu/carvel-kapp-controller/issues/535) | Early August |
-| Provide Stability | **[kapp-controller]** Version safeguards: To provide safeguards and visibility for kapp-controller and kubernetes versions. | Early August |
-| Package Author Improvements | **[kctrl]** kctrl commands for package authors. | Aug 2022 |
-| Package Author Improvements | **[ytt]** [Schema Validations](https://hackmd.io/pODV3wzbT56MbQTxbQOOKQ#Part-7-Validating-Documents) GA: Configuration authors can specify the valid range or format of the data values. | Late August |
-| Provide Stability | **[kapp-controller]** Minimize registry communications: To provide a more performant experience for users leveraging large PackageRepos. | Late August |
-
+`Last Updated: Aug 2022`
+|Theme|Feature|Stage|Timeline|
+|---|---|---|---|
+| Package Author Experience | **[kctrl]** [kctrl commands for package authors.](https://github.com/vmware-tanzu/carvel-kapp-controller/issues/632): CLI-based Package Author commands to enable  Package Authors easily create a Carvel package of their software | Build | Q3 (Jul-Sep) |
+| Package Author Experience | **[kctrl]** [kctrl dev deploy.](https://github.com/vmware-tanzu/carvel-kapp-controller/issues/2): Enable quick iteration on Package CRs and App CRs using kapp-controller CLI for local development | Build | Q3 (Jul-Sep) |
+| Package Author Experience | **[ytt]** [Schema Validations](https://hackmd.io/pODV3wzbT56MbQTxbQOOKQ#Part-7-Validating-Documents): Configuration authors can specify the valid range or format of the data values. || Q3 (Jul - Sep) |
+| Package Author Experience | **[carvel]** Carvel supports the ability to sign and verify assets (such as images, bundles, pkg/pkgr). |Awaiting Proposal| TBD |
```

Comment:
I would recommend moving any TBD-timeline items to the bottom of the table so that it's easier to follow from a chronological perspective.

Reply:
will do